spec/request/va_facilities_request_spec.rb

```diff
@@ -57,7 +67,7 @@
       expect(response).to be_success
       expect(response.body).to be_a(String)
       json = JSON.parse(response.body)
-      expect(json['data'].length).to eq(9)
+      expect(json['data'].length).to eq(10)
```

Comment:
It might be better to have a JSON schema that gets validated here instead of just the length of the data object.